clang/test/dpct/helper_files_ref/include/util.hpp

```diff
@@ -524,6 +524,23 @@ class logical_group {
   }
 };
 }
+
+/// Get the number of bytes of free and total memory on the SYCL device.
+/// \param [out] the number of bytes of free memory on the SYCL device.
+/// \param [out] the number of bytes of total memory on the SYCL device.
+void get_sycl_mem_info(size_t &free_mem, size_t &total_mem) {
+#ifdef SYCL_IMPLEMENTATION_INTEL
```

Comment:
Using __SYCL_COMPILER_VERSION instead of SYCL_IMPLEMENTION_INTEL which is not link to the feature in line 535. 
__SYCL_COMPILER_VERSION  is the build date of the dpcpp compiler from Intel.

Reply:
okay, will use __SYCL_COMPILER_VERSION in the updated PR.